jwst/stpipe/core.py

```diff
@@ -11,7 +11,10 @@
 from stpipe import Step
 from stpipe import Pipeline
 from ..lib.suffix import remove_suffix
+import logging
 
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
```

Comment:
Why is the level being set?

Reply:
Good question - it was a brainless copy-paste from a different step, but I don't think it's needed anywhere. It does exist in nearly every python file that gets imported into a _step.py file, though. Could be something to cleanse the pipeline of, in another PR.

Reply:
Ah right. There is actually an issue about this: [JP-1090](https://jira.stsci.edu/browse/JP-1090)